### PR TITLE
Include vlt metadata files

### PR DESCRIPTION
**Pull request by cculb**
@jdigger @sagarsane @masroormohammed @pgoodrich Small change to include the two required vault metadata files - vault/properties.xml and vault/definition/.content.xml

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -16,7 +16,7 @@ commons_text_version = 1.1.8
 cq_api_uber_jar_version = 6.0.0.1
 cq_version_workflow_console = 5.6.6
 felix_osgi_version = 1.4.0
-gradle_plugins_version = 2.1.0
+gradle_plugins_version = 3.0.0
```
Comment (jdigger) [inline gradle.properties]: Latest is `3.0.1`

**Comment by jdigger** [comment]
Besides the plugin version change, looks good. +1 CR

**Comment by cculb** [comment]
@jdigger thanks for the reminder. just updated to 3.0.1.

**Comment by jdigger** [comment]
Handled by TWCable/grabbit#53

**Pull request closed without merging**